### PR TITLE
Removing logging level from cli

### PR DESCRIPTION
**Pull request by adamblev**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] The PR has modified HISTORY.rst with an appropriate description of the change (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ ] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Review by Chris113113 — APPROVED** [review]
Since we've removed specified logging levels, are we going to be logging equivalent to Debug now?

**Comment by jajan17** [comment]
The logging levels were a workaround to preserve logs on the console output since the hooks intercept stdout/stderr. They don't actually control the level of verbosity for logs that go to app insights. It impacts the logging level that you see in the console and they've been changed to what it was before (info).

**Comment by scobb** [comment]
Apologies about the CI spam. I was just seeing how TravisCI works. Can ignore.

File: src/command_modules/azure-cli-ml/azure/cli/command_modules/ml/service/realtime.py [hunk 1]
```diff
@@ -448,16 +447,15 @@ def realtime_service_delete(service_name, verb, context=cli_context):
...
+                            verb, custom_ice_url, target_runtime, app_insights_logging_enabled, model, num_replicas,
                             context=cli_context):
     """Create a new realtime web service."""
 
     verbose = verb
-    if logging_level == 'none':
-        app_insights_enabled = 'false'
-        logging_level = 'debug'
-    else:
+    if app_insights_logging_enabled:
```
Comment (scobb) [inline src/command_modules/azure-cli-ml/azure/cli/command_modules/ml/service/realtime.py]: app_insights_enabled = str(app_insights_logging_enabled).lower()

**Review by scobb — CHANGES_REQUESTED** [review]
nit on boolean -> string conversion

**Review by scobb — APPROVED** [review]

**Pull request merged**